mismi-s3/test/Test/IO/Mismi/S3/Commands.hs

```diff
@@ -151,6 +151,17 @@ prop_upload d l = withLocalAWS $ \p a -> do
   r <- read a
   pure $ r === Just d
 
+prop_s3Sink :: T.Text -> Property
+prop_s3Sink t = withAWS $ \a -> do
+  let source = a { key = key a </> Key "source" }
+  let target = a { key = key a </> Key "target" }
+  let someText = "t"<>t -- make sure the text is not empty otherwise there is nothing to stream
```

Comment:
Why does this matter? We have empty files all the time - I wouldn't expect this to break.